services-geocoding/src/main/java/com/mapbox/api/geocoding/v6/V6GeocodingService.java

```diff
@@ -66,7 +66,7 @@ Call<V6Response> forwardGeocoding(
    * @param permanent     {@link MapboxV6Geocoding#permanent()}
    * @param autocomplete  {@link V6ForwardGeocodingRequestOptions#autocomplete()}
    * @param bbox          {@link V6ForwardGeocodingRequestOptions#bbox()}
-   * @param country       {@link V6ForwardGeocodingRequestOptions#country()}
+   * @param country       {@link V6StructuredInputQuery#country()}
```

Comment:
But effectively country is still taken from `V6ForwardGeocodingRequestOptions`. Is it expected?

Reply:
I assume you mean this: https://github.com/mapbox/mapbox-java/blob/05a7bfe9b3313f4c5d3a2a6f2c77922c97efc171/services-geocoding/src/main/java/com/mapbox/api/geocoding/v6/MapboxV6Geocoding.java#L87-L107

But isn't everything there taken from `V6ForwardGeocodingRequestOptions`? Why does it say other fields above taken from `V6StructuredInputQuery` like `V6StructuredInputQuery#neighborhood()`?

I guess because it assumes the `V6ForwardGeocodingRequestOptions` will be created by the builder which takes `V6StructuredInputQuery`.

If the right thing is to revert this line I changed, should we also change the others above?

Reply:
Ah, ok, I didn't notice at the first that it's a function dedicated to structure input `structureInputForwardGeocoding()`. I think we can leave as is.